python/social_bot/keybo_control.py

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys, tty, termios
+import select
+import numpy as np
+from absl import logging
+
+
+# Get command from keyboard
+class KeyboardControl:
+    """
+    This class is used to generate demonstrations from human through keyboard.
```

Comment:
Will there be a separate class to record the actual action sequence and observations?

Reply:
No. Currently the plan is that learner get teacher's demonstrations through image, as Haichao suggest. The recording is in agent's algorithm if necessary. We can also add option that use sparse observations like teacher's full states or teacher's actions for demonstrations. 
These are not implemented yet. Still need to figure out some details about the interface with algorithm. Let's do this in another PR.